src/app/sidenav/sidenav.component.ts

```diff
@@ -19,6 +19,7 @@ export class MzSidenavComponent implements AfterViewInit, OnDestroy {
   @Input() draggable: boolean;
   @Input() edge: string;
   @Input() fixed: boolean;
+  @Input() showButtonOnMobileOnly: boolean = false;
```

Comment:
I don't think that adding an input property to add the class `hide-on-large` or `show-on-large` shoulde belong in the component. You can already add those classes straight on your button without having to pass by the component.

Reply:
@jfcere 
Yes, in the case how would I do to show the menu only on the mobile?
If I do not put [fixed]="true" it will add the `show-on-large` class and if it's next to the `hide-on-large-only` class it will overwrite and will not work = /

Reply:
Like you said, I tried not to add `[fixed]="true"` on `mz-sidenav` and add `hide-on-large-only` on the button that collapse the sidenav which I thought should do the trick but I saw what you meant .... class `show-on-large` is automatically added.

Let me some time to think about it ... i'll come back to you tomorrow.

In the meanwhile ill accept the readme.md modification.

Reply:
@jfcere Thank you my friend.